src/components/AlphabetList/types.ts

```diff
@@ -23,6 +23,7 @@ export interface AlphabetListProps extends Partial<SectionListProps<IData>> {
   indexLetterStyle?: TextStyle,
   indexLetterContainerStyle?: ViewStyle,
   indexContainerStyle?: ViewStyle,
+  letterListContainerStyle?: CSSProperties,
```

Comment:
This should also be `ViewStyle` here

```suggestion
  letterListContainerStyle?: ViewStyle,
```